### DISCUSSION TITLE
a question about instrument resample

### DISCUSSION THREAD

**Discussion by zdwangjian5** [groops-devs/groops#280] [Q&A] [answered]
I want to resample a instrument file into 0.1hz. This instrument file has some arcs and a sample rate of nearly 0.1hz, some arcs may have  some  small gaps inside the arc. How to convert this instrument file into regular 0.1hz, while fill small gaps inside the arc, but will not change the big  gaps between the arcs?  Looking for your help!

**Comment by tmayerguerr** [comment, marked as answer]
Hi,
[InstrumentResample](https://groops-devs.github.io/groops/html/InstrumentResample.html)  ignores the arc structure of the file and read all epochs at once. The maximum size of gaps which will be interpolated/filled can be set with maxDataPointRange and maxExtrapolationDistance, see [InterpolatorTimeSeries](https://groops-devs.github.io/groops/html/interpolatorTimeSeriesType.html).
Hope this answers your question.
regards
Torsten

**Reply by zdwangjian5** [reply]
Thank you!
